assets/js/blocks/featured-product/block.js

```diff
@@ -212,7 +225,40 @@ class FeaturedProduct extends Component {
 		);
 	}
 
+	renderApiError() {
```

Comment:
For now it's ok, but in case we want to extend this behavior to other blocks, we could extract this function into a component.